pyproject.toml

```diff
@@ -183,7 +183,7 @@ ignore_missing_imports = true
 [tool.ruff]
 exclude = [".git", "venv*", "build", "**/__init__.py"]
```

Comment:
Hey @kuraga :)

Yeah because it does strange things here:

__init__.py example:

```
from doctr.file_utils import is_tf_available, is_torch_available

if is_tf_available():
    from .tensorflow import *
elif is_torch_available():
    from .pytorch import *  # type: ignore[assignment]
```

and if i remember it has raised circular import issues on other places :sweat_smile: